### ISSUE TITLE
Validation/Test set urls

### ISSUE THREAD

**Issue by agoel10** [mf1024/ImageNet-Datasets-Downloader#9]
Hi, 

Thanks for this incredible tool. Huge timesaver! I had one concern though. 

Is there a way to specify the downloader to download the images only from the validation/test set. Currently, I am not sure whether the images are downloaded from the training set or test set. This may be needed to evaluate a pretrained classifier on  test set images.

**Comment by michael-iuzzolino** [comment]
This tool is indeed amazing; however, I ran into the same issue that you have. I ended up using this solution: https://gist.github.com/BIGBALLON/8a71d225eff18d88e469e6ea9b39cef4

**Comment by CMCDragonkai** [comment]
There's no such thing here. The validation/test sets are determined by the ILSVRC competitions. This tool just downloads images directly from the entire repository.
